api/integrations/github/github.py

```diff
@@ -100,8 +116,10 @@ def handle_github_webhook_event(event_type: str, payload: dict[str, Any]) -> Non
         handle_installation_deleted(payload)
     elif event_type in tag_by_event_type:
```

Comment:
This is pointless, if `event_type` isn't in `tag_by_event_type` then we'll get a `KeyError` [here](https://github.com/Flagsmith/flagsmith/pull/4586/files?diff=split&w=0#diff-9571c944c3603fa5041c2bbb3bfc36614b1a102e1d22e7552b6f943c7cb24f9dR84). I assume based on that, there is no way for `event_type` to not exist in `tag_by_event_type` but we should check on this. 

If that is the case, I think this can just be changed to (and we can remove one layer of nesting):

```python 
elif action in tag_by_event_type[event_type]
```

Reply:
I checked and it was already filtered in the view code, so the check here is unnecessary. I've updated the relevant code.